java-collections-conversions-2/src/main/java/com/baeldung/modelmapper/MapperUtil.java

```diff
@@ -0,0 +1,46 @@
+package com.baeldung.modelmapper;
+
+import org.modelmapper.ModelMapper;
+import org.modelmapper.convention.MatchingStrategies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author sasam0320
+ * @description
```

Comment:
I think you should take a look about Javadoc, you have tags that don't even exist